pkg/server/runner.go

```diff
@@ -515,12 +560,12 @@ func (r *Server) Run(c context.Context) error {
 	modelClient := getModelClient(databaseDrvDialect, databaseDrv)
 
 	initOpts := initOptions{
-		K8sConfig:      k8sCfg,
-		K8sCacheReady:  make(chan struct{}),
-		ModelClient:    modelClient,
-		SkipTLSVerify:  len(r.TlsAutoCertDomains) != 0,
-		DatabaseDriver: databaseDrv,
-		CacheDriver:    cacheDrv,
+		K8sConfig:         k8sCfg,
+		K8sCtrlMgrIsReady: &atomic.Bool{},
```

Comment:
Do you think K8sMgrIsReady is enough? since the startK8sCtrlsOptions use MgrIsReady

Reply:
the `startK8sCtrlsOptions` indicates the `MgrIsReady` is the manager of K8s controllers.